### ISSUE TITLE
Source-link in docs of DelimetedFiles broken

### ISSUE THREAD

**Issue by mauro3** [JuliaLang/julia#50035]
On https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/ (this is for Julia v1.9) this backlinks to github are broken: clicking on "Edit on GitHub" (top left) and the "source" buttons on each help-box leads to a 404 on GitHub.  Maybe todo with the recent move into a separate repo (https://github.com/JuliaData/DelimitedFiles.jl)?

**Comment by LilithHafner** [comment]
I'm observing this for all the stdlibs that I checked except for Pkg.jl. I think this has been known for a while and we should really fix it. This is a great avenue for folks to help us improve documentation and the fact that the links are broken is an unfortunate obstacle in that process.

**Comment by George9000** [comment]
On a locally built julia, which is followed by `make docs`, all the links to github work in the local html documentation: julia/doc/_build/html/en/index.html.

**Comment by Moelf** [comment]
still broken: https://docs.julialang.org/en/v1.11-dev/stdlib/Unicode/#Unicode.graphemes

**Comment by LilithHafner** [comment]
I'm adding this to the 1.11 milestone because there is an open, mergable PR to fix it. If https://github.com/JuliaLang/julia/pull/47105 doesn't merge by the 1.11 branch point, I think we should merge https://github.com/JuliaLang/julia/pull/51375 as a stop-gap measure.

Maybe we should merge https://github.com/JuliaLang/julia/pull/51375 now and backport it to 1.10 as well? It should be pretty easy to revert once the more general solution is in place.

**Comment by LilithHafner** [comment]
@mortenpi, what's the timeline like for #47105 and how back portable is it? (Asking to see if merging https://github.com/JuliaLang/julia/pull/51375 is appropriate, no pressure to hurry)

**Comment by KristofferC** [comment]
I'm removing this from the milestone since this was the case in 1.9 and the PRs in https://github.com/JuliaLang/julia/issues/50035#issuecomment-1743257528 are merged.

**Comment by LilithHafner** [comment]
Yep! I just wanted to make sure those PRs merged before 1.11 if the general solution hadn't landed (which it still hasn't IIUC)
